subiquity/server/controllers/install.py

```diff
@@ -666,6 +667,17 @@ async def install(self, *, context):
             )
             raise
 
+    async def platform_postinstall(self):
+        """Run architecture specific commands/quirks"""
```

Comment:
genuine question do we want to use a context here to inform the user this is happening?

Reply:
I think that would be a good idea. At least for platforms that don't just noop.